common/src/View/InputState.h

```diff
@@ -81,6 +81,9 @@ namespace TrenchBroom {
 
             MouseButtonState mouseButtons() const;
             bool mouseButtonsDown(MouseButtonState buttons) const;
+            /**
+             * Checks whether only the given buttons are down (and no others).
+             */
             bool mouseButtonsPressed(const MouseButtonState buttons) const;
```

Comment:
If you can think of a better name for this, I'm happy to change it. I never remember which is which.

Reply:
TBH I prefer just `(mouseButtons() & query) != 0` and `(mouseButtons() == query)` rather than these helper functions. But if we keep them, could give them set-theory style names like `mouseButtonsDownEquals()` and `mouseButtonsDownContains()`